scripts/generate_dataset.py

```diff
@@ -0,0 +1,156 @@
+import pandas as pd
+import gc
+import torch
+from loguru import logger
+from skyrim.common import save_forecast
+from datetime import datetime, timedelta
```

Comment:
I suggest we import the entire module for consistency, i.e. `import datetime`

Reply:
it adds more horizontal space and you really use mostly these.

Reply:
good points on programming style here: https://docs.fast.ai/dev/style.html